odata-protocol/11.4 Data Modification.md

```diff
@@ -425,9 +425,10 @@ request that the response SHOULD include a body by specifying a
 [`return=representation`](#Preferencereturnrepresentationandreturnminimal) preference, or by
 specifying the system query options
 [`$select`](#SystemQueryOptionselect) or
-[`$expand`](#SystemQueryOptionexpand). If the service uses ETags for
-optimistic concurrency control, the entities in the response MUST
-include ETags. If a representation of the updated entity could not be constructed,
+[`$expand`](#SystemQueryOptionexpand).
+Any entity created or updated as part of the request that requires concurrency control
```

Comment:
```suggestion
Any entity that requires concurrency control
```
would be easier to implement, and easier to understand for the client. It might be tricky to determine whether an entity was "created or updated as part of the request".